stm32-modules/heater-shaker/firmware/motor_task/motor_policy.cpp

```diff
@@ -52,11 +52,12 @@ auto MotorPolicy::set_rpm(int16_t rpm) -> ErrorCode {
     }
     int16_t current_speed = get_current_rpm();
     int16_t command_01hz = rpm * _01HZ / _RPM;
-    uint16_t ramp_time = std::max(
-        static_cast<uint16_t>(
-            static_cast<int32_t>(std::abs(rpm - current_speed)) / ramp_rate),
-        static_cast<uint16_t>(1));
-    MCI_ExecSpeedRamp(hw_handles->mci[0], -command_01hz, ramp_time);
+    uint16_t ramp_time_ms =
```

Comment:
Nit: That's a lot of stuff in one statement. Could we split it up to multiple statements?